ci_framework/roles/edpm_build_images/tasks/package.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Package edpm-hardened-uefi image inside container image
+  args:
+    chdir: "{{ cifmw_edpm_image_builder_repo_path }}"
+  ansible.builtin.shell:
+    cmd: |
+      buildah bud -f ./Containerfile.image -t edpm-hardened-uefi:{{ cifmw_edpm_build_images_tag }} > {{ cifmw_edpm_build_images_artifacts }}/edpm_images/edpm_hardened_uefi_container_package.log
```

Comment:
same comment:-

1) logs should go in logs directory and not to the artifacts directory
2) Please also redirect stderr to a file.